src/Chirp.Razor/CheepContext.cs

```diff
@@ -61,6 +61,7 @@ public class Cheep
 
 public class CheepDTO
 {
+    public required int CheepId { get; set; }
```

Comment:
We might want to rename this to AuthorId, since it is the id of the author and not the id of the actual cheep.